flocker/control/test/test_model.py

```diff
@@ -1418,42 +1418,51 @@ def test_applying_removes_node(self):
             DeploymentState(nodes={node_2}))
 
 
-class NonManifestDatasetsWipingTests(SynchronousTestCase):
+class NoWipeTests(SynchronousTestCase):
     """
-    Tests for ``NonManifestDatasets.get_information_wipe()``.
+    Tests for ``NoWipe``.
     """
-    NON_MANIFEST = NonManifestDatasets(datasets={MANIFESTATION.dataset_id:
-                                                 MANIFESTATION.dataset})
-    WIPE = NON_MANIFEST.get_information_wipe()
-
     def test_interface(self):
         """
-        The object returned from ``NodeStateWipe`` implements
-        ``IClusterStateWipe``.
+        ``NoWipe`` instances provide ``IClusterStateWipe``.
         """
-        self.assertTrue(verifyObject(IClusterStateWipe, self.WIPE))
+        self.assertTrue(verifyObject(IClusterStateWipe, NoWipe()))
 
     def test_key_always_the_same(self):
         """
-        The ``IClusterStateWipe`` always has the same key.
+        A ``NoWipe`` always has the same key.
         """
-        self.assertEqual(
-            NonManifestDatasets().get_information_wipe().key(),
-            self.WIPE.key())
+        self.assertEqual(NoWipe().key(), NoWipe().key())
 
     def test_applying_does_nothing(self):
         """
-        Applying the ``IClusterStateWipe`` does nothing to the cluster state.
+        Applying the ``NoWipe`` does nothing to the cluster state.
         """
-        # Cluster has some non-manifested datasets:
-        cluster_state = self.NON_MANIFEST.update_cluster_state(
-            DeploymentState())
+        non_manifest = NonManifestDatasets(datasets={MANIFESTATION.dataset_id:
+                                                     MANIFESTATION.dataset})
+        cluster_state = non_manifest.update_cluster_state(DeploymentState())
```

Comment:
:grey_question: The above three lines are now just a construction of an arbitrary non-null cluster_state, correct? I feel like this might be more maintainable if that chunk was separated out as a global or otherwise labelled as an arbitrary cluster_state. (So future maintainers realize there is nothing special about using NonManifestDatasets in this example.

Obviously ignore if I'm misreading this.